src/features/script-load.js

```diff
@@ -12,7 +12,7 @@ systemJSPrototype.register = function (deps, declare) {
 
 systemJSPrototype.instantiate = function (url, firstParentUrl) {
   const loader = this;
-  if (url.endsWith('.json')) {
+  if (url.substr(-5) === '.json') {
```

Comment:
I guess `slice` would have been better actually?

Reply:
how so?

Reply:
Because we use it everywhere else it should have better compression in the final build.